client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/list.js

```diff
@@ -42,12 +43,13 @@ const PackageList = ( props ) => {
 		const onOpenClick = () => props.openPackage( orderId, siteId, pckgId );
 		return (
 			<div className="packages-step__list-item" key={ pckgId }>
-				<div
+				<Button
+					borderless
```

Comment:
Now that this is a button, the contrast is a bit light: 

<img width="263" alt="screen shot 2018-05-01 at 1 15 37 pm" src="https://user-images.githubusercontent.com/541093/39483947-0a086000-4d42-11e8-99d6-abc6ae12646d.png">

Can you use the `$gray-text` color for this?